certification/internal/shell/engine.go

```diff
@@ -57,12 +63,16 @@ func (e *CheckEngine) ExecuteChecks() error {
 		if err != nil {
 			log.WithFields(log.Fields{"result": "ERROR", "error": err.Error()}).Info("check completed: ", check.Name())
 			e.results.Errors = append(e.results.Errors, runtime.Result{Check: check, ElapsedTime: checkElapsedTime})
+			e.results.Status = "ERROR"
 			continue
 		}
 
 		if !passed {
 			log.WithFields(log.Fields{"result": "FAILED"}).Info("check completed: ", check.Name())
 			e.results.Failed = append(e.results.Failed, runtime.Result{Check: check, ElapsedTime: checkElapsedTime})
+			if e.results.Status == "PASSED" {
```

Comment:
There is a lot of usage of the string literals `PASSED`, `FAILED` and `ERROR` in the status and the log messages. Can we make constants for these values? This way if we need to change that for whatever reason, we can just change it in one place. My general rule of thumb is if I am using the value more than once, I make a constant for it.